src/plugins/vis_types/xy/public/convert_to_lens/configurations/index.ts

```diff
@@ -181,7 +182,11 @@ function getReferenceLineLayers(
           axisMode: getYAxisPosition(yAxis?.position ?? 'left'),
           color: thresholdLineConfig.color,
           lineWidth: thresholdLineConfig.width !== null ? thresholdLineConfig.width : undefined,
-          lineStyle: getLineStyle(thresholdLineConfig.style),
+          lineStyle:
```

Comment:
Why inlining this from the helper?

Reply:
We don't support dot-dashed type, `getLinseStyle` can return this.